backend/src/question/question.service.ts

```diff
@@ -21,20 +39,48 @@ export class QuestionService {
    * @return  Array of QuestionSummary with the relevant fields
    */
   async getAllSummaries() {
-    const res: QuestionSummaryFromDb[] =
-      await this.prisma.questionSummary.findMany({
-        select: QUESTION_SUMMARY_SELECT,
-      });
+    const summaries = await this.cache.getKeyInNamespace<
+      QuestionSummaryFromDb[]
+    >([NAMESPACES.QUESTIONS], QUESTION_SUMMARIES);
```

Comment:
think it's good practice to create a utility function for both getting and setting values to the cache, similar to what's done in the room service